src/ConstantProduct.sol

```diff
@@ -86,12 +81,11 @@ contract ConstantProduct is IConditionalOrderGenerator, IERC1271 {
     bytes32 public immutable solutionSettlerDomainSeparator;
 
     /**
-     * @notice It associates every order owner to the only order hash that can
-     * be validated by calling `verify`. The hash corresponding to the constant
-     * `EMPTY_COMMITMENT` has special semantics, discussed in the related
-     * documentation.
+     * @notice The only order hash that can be validated by calling `verify`.
+     * The hash corresponding to the constant `EMPTY_COMMITMENT` has special
+     * semantics, discussed in the related documentation.
      */
-    mapping(address => bytes32) public commitment;
+    bytes32 public commitment;
```

Comment:
Nice. Knowing that this was going to be a discrete contract per AMM, having this `mapping` was always confusing when reviewing :sweat_smile: